lectures/heavy_tails.md

```diff
@@ -78,6 +78,12 @@ We can see this when we plot the density and show a histogram of observations,
 as with the following code (which assumes $\mu=0$ and $\sigma=1$).
 
 ```{code-cell} ipython3
+---
+mystnb:
+  figure:
+    caption: Histogram of observations
```

Comment:
@jstac should the be `draws` instead of `observations`?

Reply:
I think either is fine.